src/bin/task-get-blocks.rs

```diff
@@ -0,0 +1,140 @@
+use std::cmp::min;
```

Comment:
I think this filename better be on snake_case.
https://rust-lang.github.io/api-guidelines/naming.html